sdk/python/feast/infra/provider.py

```diff
@@ -134,6 +151,14 @@ def ingest_df(
         """
         pass
 
+    def ingest_dfs(
```

Comment:
since we are creating this new ingest flow, maybe we can explore using some other data type like pyarrow data which is also used by beam i believe

Reply:
yeah, we can update later